x-pack/plugins/apm/public/components/app/settings/agent_explorer/agent_list/index.tsx

```diff
@@ -153,6 +163,51 @@ export function getAgentsColumns({
         />
       ),
     },
+    ...(latestAgentVersionEnabled
+      ? [
+          {
+            field: AgentExplorerFieldName.AgentLastVersion,
+            name: (
+              <EuiToolTip
+                content={i18n.translate(
+                  'xpack.apm.agentExplorerTable.agentLatestVersionColumnTooltip',
+                  {
+                    defaultMessage: 'The latest released version of the agent.',
+                  }
+                )}
+              >
+                <>
+                  {i18n.translate(
+                    'xpack.apm.agentExplorerTable.agentLatestVersionColumnLabel',
+                    { defaultMessage: 'Latest Agent Version' }
+                  )}
+                  &nbsp;
+                  <EuiIcon
+                    size="s"
+                    color="subdued"
+                    type="questionInCircle"
+                    className="eui-alignCenter"
+                  />
+                </>
+              </EuiToolTip>
+            ),
+            width: '10%',
+            align: 'center',
+            truncateText: true,
+            render: (
+              _: any,
+              { agentName, latestVersion }: AgentExplorerItem
+            ) => (
+              <AgentLatestVersion
+                agentName={agentName as AgentName}
```

Comment:
no need to cast to `AgentName` here either I think

Reply:
Addressed in [7b2cefb](https://github.com/elastic/kibana/pull/153643/commits/7b2cefb4943bce8e0da6cb80b547d1f3a0d48db4).